### DISCUSSION TITLE
The submission of private warehouses is not counted

### DISCUSSION THREAD

**Discussion by EternityJune25** [anuraghazra/github-readme-stats#3712] [Q&A] [answered]
Added parameters as required in the document, but it doesn't seem to work (private warehouse submission is not counted)🤔

**Comment by qwerty541** [comment, marked as answer]
Hey, @EternityJune25. Our documentation says that displaying private contributions require [deploying you own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) with your own GitHub API token. As I understand, you was confused by outdated chinese documentation translation, query string parameter `&count_private=true` is no more supported. We are going to remove documenattion translation in nearest future since our team is not able to maintain them. Please use english version.

![Снимок экрана от 2023-11-28 19-28-34](https://github.com/anuraghazra/github-readme-stats/assets/53787217/3016c5c0-24f0-48e8-b9e3-b57f37a39ff9)

![Несохранённое изображение 1](https://github.com/anuraghazra/github-readme-stats/assets/53787217/714d6b1e-1643-4b28-ab7c-9be6d65c3a6f)

**Reply by EternityJune25** [reply]
Thank you. Problem solved.

**Reply by qwerty541** [reply]
Glad to help. Please mark this discussion as answered.
